setup.py

```diff
@@ -45,7 +45,7 @@
 
 logger = logging.getLogger(__name__)
 
-version = '2.3.4.post35'
+version = '2.3.4.post36'
```

Comment:
@luisglft , can you change the version to a pre-release version (I forget exactly the format we use, but we have something after the post# to reflect that it's not a version that's been finalized yet)? Or did you have a reason not to do that?

Reply:
No reason in particular, just changed to version = '2.3.4.post36-debug0'